node/service/src/chain_spec/pangoro.rs

```diff
@@ -35,7 +34,8 @@ use darwinia_staking::StakerStatus;
 use drml_common_primitives::*;
 use pangoro_runtime::*;
 
-pub type ChainSpec = sc_service::GenericChainSpec<GenesisConfig>;
+// pub type ChainSpec = GenericChainSpec<GenesisConfig, Extensions>;
```

Comment:
unused comment?

Reply:
It will be activated in the next version

Reply:
Better to add some comments?

Reply:
NP